src/components/NoteManager/components/NotesActions.tsx

```diff
@@ -73,8 +90,18 @@ export function NotesActions() {
         </button>
         <button onClick={onImport}>📂 import</button>
         <button onClick={handleExport}>💾 export</button>
+        <button
+          data-tooltip-id="delete-tooltip"
+          data-tooltip-content={confirmDelete ? "Yes, delete" : "Delete all notes"}
+          data-tooltip-place="bottom"
+          disabled={confirmDeleteDisabled}
+          onClick={deleteNotes}
+        >
+          {confirmDelete ? "Are you sure?" : "🗑 delete"}
```

Comment:
```suggestion
          {confirmDelete ? (secondsLeft >0 `Wait (${secondsLeft})` : 'Confirm Removal' ) : "🗑 delete"}
```

Could you please add one more thing. A common pattern, when the button is inactive because we want the user wait a bit is to display the number of seconds left.

Either let's display on a button or in the toolip.